src/index.js

```diff
@@ -1,11 +1,48 @@
 /*eslint no-unused-vars: "off"*/
-import React from 'react';
+import React, { Component } from 'react';
 import { render } from 'react-dom';
+import ToDoList from './components/toDo';
 // You'll want to import the necessary components you want the App component to render
 
-const App = () => (
-   {/* Replace this div with whatever components you want the App root component to render */}
-   <div>Hello World!</div> 
-);
+// const App = () => (
```

Comment:
Please remove commented out code, when submitting PRs.

Reply:
I will be sure to remove the comments out of my code from now on. I don't fully grasp what's going on but am going over your videos right now to try to nail it down in my head.